composer.json

```diff
@@ -6,7 +6,7 @@
     ],
     "type": "neos-package-collection",
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
```

Comment:
needs to be changed in all the single packages too.

Reply:
It is changed, no? What did I miss?

Reply:
Wasn't the case in the first commit :)